crates/tokenizer/src/lexers.rs

```diff
@@ -381,7 +406,7 @@ fn handle_operator<'a>(mut step: Step<'a>, token: &Token) -> Option<Step<'a>> {
             Some(step)
         }
         Token::TripleQuoteOpen => {
-            step.mode = LexGrammarMode::MultilineString;
+            step.set_mode(LexGrammarMode::MultilineString);
```

Comment:
set_mode, changes previous mode to current mode and replaces current mode. We only want this when parsing single line strings as they can be inside multiline strings. But this shouldn't be a problem logically